lib/workers/repository/dependency-dashboard.ts

```diff
@@ -160,7 +160,7 @@ export async function ensureDependencyDashboard(
   if (awaitingSchedule.length) {
     issueBody += '## Awaiting Schedule\n\n';
     issueBody +=
-      'These updates are awaiting their schedule. Click on a checkbox to ignore the schedule.\n';
+      'These updates are awaiting their schedule. Click on a checkbox to get a update now.\n';
```

Comment:
Maybe

```suggestion
      'These updates are awaiting their schedule. Click on a checkbox to get an update now.\n';
```
?

Reply:
Good catch! I've pushed a commit that fixes this. 😄